memory/ndctl.py

```diff
@@ -333,6 +333,24 @@ def test_namespace_reconfigure(self):
         if failed_vals:
             self.fail("New namespace unexpected change(s): %s" % failed_vals)
 
+    def test_check_namespace(self):
+        """
+        Verify metadata for sector mode namespaces
+        """
+        self.enable_region()
+        region = self.get_json(short_opt='-R')[0]
+        self.disable_namespace()
+        self.destroy_namespace()
+        self.log.info("Creating sector namespace using %s", region)
+        self.create_namespace(region=region, mode='sector')
+        ns_sec = self.get_json()[0]
```

Comment:
This assignment can be skipped

Reply:
@narasimhan-v pushed with the change, Thanks.